cypress/e2e/circulation-log/same-date-and-time-format-as-fee-fine-details-page.cy.js

```diff
@@ -0,0 +1,117 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import { TestTypes } from '../../support/dictionary';
+import devTeams from '../../support/dictionary/devTeams';
+import permissions from '../../support/dictionary/permissions';
+import servicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
```

Comment:
Use PascalCase for imported fragments to match the code style

Reply:
Got it, done